frontend/src/scenes/data-warehouse/new/dataWarehouseTableLogic.tsx

```diff
@@ -45,6 +45,16 @@ export const dataWarehouseTableLogic = kea<dataWarehouseTableLogicType>([
         setDataTableQuery: (query: DataTableNode) => ({ query }),
     }),
     loaders(({ props }) => ({
+        tables: [
+            [] as DataWarehouseTable[],
+            {
+                loadTables: async () => {
+                    return await api.dataWarehouseTables
+                        .list({ exclusion_pattern: 'https://posthog-s3-datawarehouse' })
```

Comment:
@Gilbert09 Let me know if this would be the best way before a more drastic change committing a column to the db...